pkg/api/client.go

```diff
@@ -54,6 +55,14 @@ type httpClient struct {
 	timeout    time.Duration
 }
 
+type ConfigAgent struct {
+	EnvironmentName string
+	IsDocker        bool
+	AgentName       string
+}
+
+var CfgAgent ConfigAgent
```

Comment:
Maybe make this and the struct private then add a func to receive the 3 values to create it?

You may also want to create a default one with an init() func.